### PR TITLE
interfaces: disable auto-connect in libvirt interface

### PR DESCRIPTION
**Pull request by mvo5**
Sorry my fault for merging this before it had a proper review from the security team.

### CODE REVIEW

File: interfaces/builtin/libvirt.go [hunk 1]
```diff
@@ -43,6 +43,7 @@ func NewLibvirtInterface() interfaces.Interface {
 		connectedPlugAppArmor: libvirtConnectedPlugAppArmor,
 		connectedPlugSecComp:  libvirtConnectedPlugSecComp,
 		reservedForOS:         true,
-		autoConnect:           true,
+		// cannot auto-connect, it grants too much power
+		autoConnect: false,
```
Comment (jdstrand) [inline interfaces/builtin/libvirt.go]: Thanks for making this change. Can you add a test for autoconnect? Something like:

```
func (s *LibvirtInterfaceSuite) TestAutoConnect(c *C) {
	c.Check(s.iface.AutoConnect(), Equals, false)
}
```

**Comment by jdstrand** [comment]
Once this test is added and the testsuites pass, +1 (though I will have other comments to address later that can be fixed in a future commit).

**Comment by mvo5** [comment]
Hm, fails in the reboot test https://travis-ci.org/snapcore/snapd/jobs/158497543#L1696

**Comment by zyga** [comment]
I'm not familiar with the fistboot logic but the failure looks unrelated:
```
++ cat firstBoot
+ snap install --dangerous /var/lib/snapd/snaps/ubuntu-core_/et.snap
error: cannot open: "/var/lib/snapd/snaps/ubuntu-core_/et.snap"
```

**Pull request merged**